src/content/docs/blog/2025/06/30/mid-year-update/index.md

```diff
@@ -0,0 +1,185 @@
+
+Until these features are implemented, however, maintaining the AerynOS repository will remain somewhat human resource intensive. This is the main reason why the repository is consciously being kept "small" for now, with us deliberately focusing on having packages that will help developers and contributors to improve AerynOS, while still delivering a nice Daily Driver experience.
+
+Until the new features are implemented, this will necessarily be a balancing act between maintaining the package repository so it doesn't go stale vs. having the development time to implement the new features.
+
+### How did you test it?
+
+Aside from porting the infra code to Rust, proper testing was required to yield confidence that packages were both successfully built on the new infra and that they worked as expected. 
+
+The end goal was to prove that we were able to rebuild the full AerynOS recipes repository (currently at ~950 recipes) from start to finish without infra-related build errors on the new infra.
+
+To enable the rebuild, ermo set up a distributed build cluster of four builders of varying hardware specifications. A separate branch of the 'recipes' repository was created, and was used to test both the Rust infra and to land packages for internal testing without them being seeded to user installs.
+
+In addition, compared to the old infra, we made it simpler to add new avalanche build agents to the build cluster, thus making it very simple to scale out our build cluster as required.
+
+To summarise the infra Rust re-write and testing effort, we have:
+
+- Completed more than 3k recipe builds
+- Deployed the new Rust infra on the AerynOS builders and continue to use it on ermo's build cluster
+- Validated that the new infra code is more stable and performant at runtime than the previous DLang version
+
+### How did testing add value?
+
+The full rebuild of the recipes repository has also served to ensure ABI sanity for dependencies. Additionally, we can now say that at this point in time, the whole AerynOS repository is known to be buildable and works with all the latest toolchains.
+
+A special thanks goes to Reilly Brogan, who worked diligently with ermo to not only drive the rebuild process, but also to ensure that some longstanding repository issues were corrected as part of the rebuild process. 
+
+During this process, we have delivered updates to our os-tools (boulder and moss), toolchains and build systems. A selection of the updates and addition include (but is certainly not limited to):
+
+- Linux 6.14.11 (6.15.x on the way)
+- LLVM 20.1.7
+- GCC 15.1.1
+- Rust 1.88.0
+- Golang 1.24.4
+- Mesa 25.1.4
+- GNOME 48.2
+- COSMIC 1.0.0_alpha7
+- Sway 1.11
+- Firefox 140.0.1
+- Thunderbird v139.0.2
+- Uutils-coreutils 0.1.0
+- Nodejs 22.14.0
+- Wine 10.8
+- Distrobox added at v1.8.1.2
+- Exfatprogs added at v1.2.9
+- Fzf added at v0.62.0
+- Kitty added at v0.41.1
+- Waybar added at v0.12.0
+
+## A New Package Repository
```

Comment:
Again unusual capitalisation here. Google Style Guide (or whatever it's called) used in Solus says only first word capitalised in headings.